includes/scroll-button.html

```diff
@@ -0,0 +1,4 @@
+<div href="" class="arrow-for-scrolling-top" id="js-scroll-top">
+    <span class="arrow"></span>
+</div>
```

Comment:
Can we make this button a semantic `<button>`? That way it follows things like tab ordering, hover/focus/click behavior - all really helpful for screen readers and keyboard controls!

```suggestion
<button id="js-scroll-top" class="arrow-for-scrolling-top">
  {% octicon "arrow-up" %}
</button>
```

This might require some additional style tweaking, but it's worth it!

I also used an [Octicon](https://primer.style/octicons/) here, so that we use a consistent visual language.

Reply:
@JasonEtco 
Thank you for your advices🙇‍♂️
I rewrite 'div' to 'button' and changed to use octicon!